src/psp-migration.md

```diff
@@ -0,0 +1,375 @@
+# How to migrate from Kubernetes PSPs to Kubewarden policies?
+Kubewarden can be used to replace all deprecated [PodSecurityPolicy](https://kubernetes.io/docs/concepts/security/pod-security-policy/). Different from the PSPs, Kubewarden has separated
+policies to achieve the same goal of different fields from the PSPs. The policies to cover each of the PSP configuration fields can be found below in the [mapping table](#mapping-kuberwarden-policies-to-psp-fields).
```

Comment:
Nit: I would drop this line to be fair, and assume that they can find it on their own (my personal preference is as shorter docs as possible, sometimes to the extreme..).

Reply:
We can drop it and have a pre-requisites section linking the installation section to this doc instead of an explicit mention.

Reply:
As a how-to guide, I guess we can assume that user know how to install Kuberwarden and has a running cluster ready to be used. Does it make sense?

Reply:
I'm happy assuming, and I suppose they will have an "installing Kubewarden" on the left of the page on the sections at some point.